src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/HiddenFieldCheck.java

```diff
@@ -125,6 +125,24 @@
  * <pre>
  *  &lt;module name=&quot;HiddenField&quot;/&gt;
  * </pre>
+ * <pre>
+ * public class SomeClass {
+ *
+ *   private String field;
+ *
+ *   public void method(String param) { // OK
+ *       String field = param; // violation, 'field' variable hides 'field' field
```

Comment:
please add in this case ctor and please use this example block in all examples.

code is the same ... only config is changing .. user see a difference by comments

Reply:
check check made some changes but i didn't touch last config example abstract one.